metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -39,6 +39,15 @@ object ClientCommands {
        |""".stripMargin
   )
 
+  val RunCode = Command(
+    "metals-scala-run",
+    "run",
```

Comment:
Maybe something nicer? Could we do a unicode character if unicode is enabled? something like `➡ Run main` 

And `runs code` seems a little lackluster

Reply:
or `▶ run`

Reply:
> And runs code seems a little lackluster
yup, would "Runs main method" be better? I do not like unnecessary words, as you can see ;P

> Could we do a unicode character (...) ?
Up to a debate (@olafurpg ?) I do not have an opinion here

Reply:
I'd go with `Run`, using unicode seems like cheating and might not play well if Code adds an icon itself, I guess (I don't know much about the protocol so idk if that sort of change could happen, but it might be worth considering :) )

Reply:
I agree with @kubukoz, "Run" is good.